### PR TITLE
Api to get invitations info

### PR DESCRIPTION
**Pull request by Shevil27**
An API that gets invitation details using the code invitation code.

### CODE REVIEW

File: packages/base/src/auth/system.ts [hunk 1]
```diff
@@ -26,6 +27,11 @@ export type AuthOptions = {
    */
   usePassword: boolean;
 
+  /**
+   * If the app should use invitation authentication.
+   */
+  useInvitation: boolean;
```
Comment (mistyharsh) [inline packages/base/src/auth/system.ts:33]: Where is this flag required? If not, please remove it.

File: packages/base/src/auth/invitation.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    //     return c.json({ value: result });
+    // })
+
+    app.get('/invitations/:code', async (c) => {
+        const code = c.req.param('code');
+        const request = await findInvitationByCode(c.var.authContext.db, code);
+
+        return c.json(request);
+    })
+}
```
Comment (mistyharsh) [inline packages/base/src/auth/invitation.ts:37]: Add a newline character.

File: packages/base/src/auth/invitation.ts [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+import type { HonoAuthApp } from './type.js';
+
+const InvitationClaimDTO = z.object({
+    code: z.string(),
+    password: z.string(),
+  });
+
+export async function addInvitationStrategy(app: HonoAuthApp): Promise<void> {
+    // app.post('/claim-invitation', async (c) => {
```
Comment (mistyharsh) [inline packages/base/src/auth/invitation.ts:13]: Remove this not-required commented code.

**Review by mistyharsh — CHANGES_REQUESTED** [review]
Please address the comments.

File: packages/base/src/auth/invitation.ts [hunk 4]
```diff
@@ -0,0 +1,19 @@
+import { claimInvitation } from '@webf/auth/context';
+import { findInvitationByCode } from '@webf/auth/dal';
+import { z } from 'zod';
+
+import type { HonoAuthApp } from './type.js';
+
+const InvitationClaimDTO = z.object({
+    code: z.string(),
```
Comment (ubcharu) [inline packages/base/src/auth/invitation.ts:8]: Fix indentation in this file.

**Review by mistyharsh — APPROVED** [review]

**Pull request merged**